scripts/deploy-dev-ec2.sh

```
@@ -0,0 +1,187 @@
+
+echo "✅ IAM setup complete."
+echo ""
+sleep 5 # Allow time for IAM changes to propagate
+
+# --- Security Group ---
+echo "🌐 Creating Security Group..."
+if ! aws ec2 describe-security-groups --group-names "$SECURITY_GROUP_NAME" >/dev/null 2>&1; then
+  SECURITY_GROUP_ID=$(aws ec2 create-security-group \
+    --group-name "$SECURITY_GROUP_NAME" \
+    --description "Allow SSH and App traffic for FixFox Dev" \
+    --vpc-id $(aws ec2 describe-vpcs --filters "Name=is-default,Values=true" --query "Vpcs[0].VpcId" --output text) \
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)